docs/examples/bouncer-bucket.md

```diff
@@ -0,0 +1,181 @@
+
+### Creating the bounce
+
+A game update interval (the **onUpdateInterval** function) checks to see if any balls have reached the bottom of the screen. If they have, their velocity is set to negative to make them go back up. The game physics knows to make the acceleration invert so the ball will slow down as it moves upward. It will actually stop at the value of **y** were it started. But at that point, the game physics changes the direction of acceleration and the ball begins to drop again.
+
+This is a nice effect but we know that in real life a ball will go to a lower height each time it bounces due to the loss of energy. We can simulate this by adding a little bit more acceleration to the ball each time it bounces. This creates a decay effect causing the bounce to peak at a lower point each time. In the game, the decay acceleration added to the current acceleration is `20`.
+
+Since the came can have multiple balls bouncing at the same time, their sprites are kept in an array so the game update interval can scan for which ones have hit the bottom of the screen.
+
+### Catching the balls
+
+The balls are caught in a bucket sprite. The bucket is moved with the left and right buttons to make it line up under a dropping ball. The **x** position of the bucket is adjusted in an **onUpdate** function by adding the button change, or _delta_, value of **dx**. There is a 3 pixel target width for the ball to enter the bucket. So, the **x** value of the ball can only be `1` pixel on either side of the **x** value of the bucket to make a catch and score points.
+
+To make it a little move difficult to score points, the bucket must move to a new position every `3` seconds. Otherwise, the player could just leave the bucket in one place and wait for balls to randomly drop in.
+
+When the balls hit the bucket but miss the target spot, they are bounced away from the side of the bucket they contacted.
+
+The bucket catching code uses the **onOverlap** event to detect when a ball sprite reaches the bucket sprite.
+
+### Dropping balls
+
+Each ball is dropped at a random location in the upper corner of the screen. The player can't expect a ball to drop at the same place every time. When a ball bounces off the screen it's destroyed and replaced by another ball whose type is randomly chosen.
+
+### Playing the game
+
+There are two modes of play: single ball and multiple balls. Press the **A** button to drop a single ball and press the **B** to drop `10` balls at once. The single ball mode runs for `45` seconds and the multiple ball mode runs for `30` seconds. The player trys to catch as many balls as possible while the time counts down.
```

Comment:
trys -> tries?

Reply:
Tries, yes. How'd that happen? It's like "would you like frys with your meal".